### PR TITLE
Unify rendering Calendar

### PR DESCRIPTION
**Pull request by epetrow**
closes: https://github.com/telerik/kendo-themes-private/issues/213

Design: https://www.figma.com/file/8bWKUiXYan6Xk5d6Xl2NcO/%23556-Calendar-and-MultiView---Rendering?type=design&node-id=0%3A1&mode=dev

> Some context around the implemented changes:
> - Week number and Week days cell font size - changed to `kendo-font-size-sm` for all themes except Fluent were we will stick with their guidelines i.e. all cells have identical font size of 12px
> - `k-today` font-weight - increased to bold as per design suggestion
> - Week header and Week number column cell text color will remain `$kendo-subtle-text` as using `$kendo-disabled-text` does not provide enough contrast
> - Paddings for Classic Calendar bodies, headers and footers - unified across all themes as per design suggestion

### CODE REVIEW

File: packages/html/src/calendar/templates/calendar-normal.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const CalendarNormal = ({ showOtherMonth, ...props }: any) => (
         children={
             <>
                 <tr className="k-calendar-tr">
-                    <CalendarCell text="30" showOtherMonth={showOtherMonth} otherMonth weekend />
+                    <CalendarCell text="30" showOtherMonth={showOtherMonth} empty={!showOtherMonth}/>
```
Comment (zhpenkov) [inline packages/html/src/calendar/templates/calendar-normal.tsx:8]: This cell is missing _weekend_.

File: packages/html/src/calendar/templates/calendar-decade.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const CalendarDecade = (props: any) => (
         children={
             <>
                 <tr className="k-calendar-tr">
```
Comment (zhpenkov) [inline packages/html/src/calendar/templates/calendar-decade.tsx:7]: Update with `<CalendarTableRow>`.  Applies to other files as well.

File: packages/html/src/calendar/tests/calendar-infinite-long-names.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { CalendarInfiniteNormal } from '../index';
+
+export default () => (
+    <>
+        <div id="test-area" className="k-grid">
```
Comment (zhpenkov) [inline packages/html/src/calendar/tests/calendar-infinite-long-names.tsx:5]: Typo in `test-area` element, the className should be `k-d-grid`. 
`k-grid` breaks the calendar cells in the visual tests.

File: tests/calendar/calendar-modern-size.html [hunk 4]
```diff
@@ -14,8 +14,8 @@
         <span>large</span>
         <section>
             <div class="k-calendar k-calendar-sm">
-                <div class="k-calendar-header">
-                    <button class="k-calendar-title k-button k-button-sm k-button-flat k-button-flat-base k-rounded-md">
+                <div class="k-calendar-header k-hstack">
```
[the remark this answers is not in the document]
Comment (epetrow) [reply tests/calendar/calendar-modern-size.html:17]: It was, but we don't actually need it, so I'll remove it

**Review by zhpenkov — APPROVED** [review]
LGTM!

**Review by inikolova — APPROVED** [review]

**Pull request merged**